src/modality-specific-files/task-events.md

```diff
@@ -309,3 +309,97 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
     "VisionCorrection": "lenses"
 }
 ```
+
+### Continuously-sampled, stimulus-related signals
+
+!!! example "Example datasets"
+
+      The following [BIDS-Examples][] showcase stimulus-related signals:
+
+      -   ["synthetic" example dataset][synthetic-example].
+
+      The following examples include these type of signals and may be used
+      (with caution, because they are not "official" examples)
+      as a reference when curating a new dataset:
+
+      -   [*Trial timing for multivariate pattern analysis*][ds000238]
```

Comment:
OK I just checked and it seems like the stim.tsv.gz of this dataset should be a beh.tsv

Also the error thrown by openneuro validation seems wrong: 
https://openneuro.org/datasets/ds000238/versions/00002

```
Location:

/sub-01/beh/sub-01_task-test_stim.tsv.gz
Reason:

You should define 'ArterialSpinLabelingType' for this file. 'ArterialSpinLabelingType' can be CASL, PCASL, PASL.
```

Maybe another dataset would be better.

Reply:
Good catch. It doesn't make any sense to point at an example we know is a bad example.